sdk/tests/android/testapp/build.gradle

```diff
@@ -57,12 +58,6 @@ android {
         sourceCompatibility = 1.8
         targetCompatibility = 1.8
     }
-    ndkVersion '23.1.7779620'
```

Comment:
Trivial:
According to [docs](https://developer.android.com/studio/projects/install-ndk#apply-specific-version), `android.ndkVersion` is the right place, so probably `android.defaultConfig.ndkVersion` should be removed.
Also, a question: why do we use NDK in this repo? The native code is in `aws-crt-java`, so NDK is required only there, right?

Reply:
I think I've removed all references to NDK in the SDK.